src/implementations/twig/components/carousel/carousel.story.js

```diff
@@ -21,7 +22,7 @@ const getArgTypes = () => {
       name: 'width',
       type: 'select',
       description: `The media container extends to the whole viewport by default when outside the grid,
-        if it's inside it can still be extended by adding class .ecl-page-banner--full-width`,
+        if it's inside it can still be extended by adding class .ecl-banner--full-width`,
```

Comment:
there is a default type "radio" in this control, considering that this is not related to a param in the template i would simply remove the table with the defaults